builder.go

```diff
@@ -153,13 +153,33 @@ func (p *MailBuilder) GetBCC() []mail.Address {
 // ReplyTo returns a copy of MailBuilder with this name & address appended to the To header.  name
 // may be empty.
 func (p MailBuilder) ReplyTo(name, addr string) MailBuilder {
-	p.replyTo = mail.Address{Name: name, Address: addr}
+	if len(addr) > 0 {
+		p.replyTo = append(p.replyTo, mail.Address{Name: name, Address: addr})
+	}
+	return p
+}
+
+// ReplyToAddrs returns a copy of MailBuilder with the new reply to header list. This method only
+// has an effect if the Send method is used to transmit the message, there is no effect on the parts
+// returned by Build().
+func (p MailBuilder) ReplyToAddrs(replyTo []mail.Address) MailBuilder {
+	p.replyTo = replyTo
 	return p
 }
 
-// GetReplyTo returns the stored replyTo header.
+// GetReplyTo returns the first stored replyTo header.
```

Comment:
Given enmime is pre-1.0, I'd prefer changing the API contract here and just having GetReplyTo return the slice instead so that it matches GetTo & GetCC

Reply:
Hi @jhillyerd 

If you're going to break API contract in next release, do you accept PR to keep original header cases (lower / upper)?
https://github.com/jhillyerd/enmime/pull/283#issuecomment-1464835088

Reply:
@iredmail yeah, might as well do that at the same time.